itest/datasets/case03/partition/case03/RefDeepSrcMatch/chunk_info.json

```diff
@@ -0,0 +1,50 @@
+{
+	"chunkStats": {
```

Comment:
Does this use TABs for alignment? I think we prefer spaces for all file formats.

Reply:
This file is extracted from qserv integration test, so it should be fixed in qserv repository (ping @iagaponenko @fritzm )